### PR TITLE
added ../ to csv path

### PR DESCRIPTION
**Pull request by honnsey**

### CODE REVIEW

File: data_scientist_skills/data_engineering.py [hunk 1]
```diff
@@ -19,13 +24,15 @@ def level_encoder(df):
...
+    analyst, engineer and scientist based on job title
     '''
 
     titles = ['analyst', 'engineer', 'scien']
 
+    # Include "encoded" in column name, as keywords extracted from job
+    # description may include words in titles
     for title in titles:
-        df[title] = df['cleaned_title'].str.contains(title)\
+        df[f'encoded {title}'] = df['cleaned_title'].str.contains(title)\
```
Comment (RemoteSalad) [inline data_scientist_skills/data_engineering.py]: This column should have an underscore instead of space

**Review by RemoteSalad — CHANGES_REQUESTED** [review]
This looks good, I just think the dynamically created titles in the job_type_encoder method should have an underscore '_' instead of a space ' '

File: data_scientist_skills/data_engineering.py [hunk 2]
```diff
@@ -76,3 +83,70 @@ def process_salary_estimate(dataframe, only_mean = True):
...
+
+    def get_level(row):
+        '''Return job level: junior, mid-level or senior'''
+        return levels[np.argmax(row.values)][1]
+
+    def get_stream(row):
+        '''Return stream: scientist, analyst, engineer'''
+        return streams[np.argmax(row.values)]
+
+    Category = df[streams].apply(get_stream, axis= 1) \
```
Comment (sneaky-hobbitses) [inline data_scientist_skills/data_engineering.py:145]: KeyError: "None of [Index(['encoded analyst', 'encoded engineer', 'encoded scien'], dtype='object')] are in the [columns]"

No columns with these names, do we need to rename in previous function? Or just keep names without 'analyst'?

File: data_scientist_skills/data_engineering.py [hunk 3]
```diff
@@ -76,3 +83,70 @@ def process_salary_estimate(dataframe, only_mean = True):
...
+    return df
+
+def get_position(df):
+    '''
+    Return the Category (level and stream) as a Series
+    Use to visualise dataframe - to compare with unsupervised model results
+    '''
+
+    levels = [('title', 'junior'), ('title', 'senior'), ('title', 'mid-level')]
+    streams = ['encoded analyst', 'encoded engineer', 'encoded scien']
```
Comment (RemoteSalad) [inline data_scientist_skills/data_engineering.py]: this should also have '_' instead of space ' '

**Review by sneaky-hobbitses — APPROVED** [review]

**Review by RemoteSalad — APPROVED** [review]

**Review by sneaky-hobbitses — APPROVED** [review]

**Pull request merged**